docs/install/ubuntu.rst

```diff
@@ -7,3 +7,49 @@ Ubuntu build script
 Scripts for installing PyNE and all its dependencies from scratch on Ubuntu
 14.04 - 15.04 are found `here
 <https://github.com/pyne/install_scripts/>`_.
+
+In this repository, you will find both bash scripts and dockerfiles for Ubuntu.
+These scripts will install PyNE and its dependencies to either your device
+or an `image <https://docs.docker.com/get-started/#images-and-containers>`_. Docker
+allows users to build these images and to operate them separate from the rest of
+your device.
+
+You can download Docker `here <https://docs.docker.com/get-docker/>`_.
+
+
+-------------------
+Bash Scripts (*.sh)
+-------------------
+
+The script used to install PyNE should correspond
+to the user's operating system and version.
+
+The intention of these
+scripts is that PyNE can be ready to use on a clean install of any of
+the supported operating systems. Furthermore, the user should choose either
```

Comment:
Can we link "supported operating systems" to the page that lists this? (or list them here if there is no page)

Reply:
Yes we could, however that page still needs to be updated. I'll add those updates to this PR, and then link it here